data/statuses.yaml

```diff
@@ -37,6 +37,26 @@
     to correct it.
 
     If this is a false positive, file a portingdb bug.
+- ident: py2mispackaged
+  name: Py2Mispackaged
+  abbrev: ''
+  color: ''
+  term: ''
+  weight: 8
+  description: Python 2 subpackage is incorrectly named
+  rank: 8
+  instructions: |
+    The Python 2 subpackage does not follow the naming policy.
+- ident: py2done
+  name: Py2Done
+  abbrev: ''
+  color: ''
+  term: ''
+  weight: 7
+  description: Python 2 subpackage is correctly named
+  rank: 7
+  instructions: |
+    The Python 2 subpackage follows the naming policy.
```

Comment:
Could this be done with the usual `mispackaged` and `released` statuses? It seems they wouldn't clash, since these are in a different field. Or is there a special reason to add new ones?